cob_omni_drive_controller/src/param_parser.cpp

```diff
@@ -20,6 +20,10 @@
 #include <angles/angles.h>
 #include <XmlRpcException.h>
 
+#include <tf2/LinearMath/Quaternion.h>
```

Comment:
These heades are available from `tf` as well.

Reply:
tf2::Vector3 tf2::Transform tf2::Quaternion  is not part of tf/tf.h
using tf/tf.h will also add dependency to `geometry_msgs/twist` which I like to avoid

Reply:
We are depending on `geometry_msgs` and `tf` anyway..

Reply:
ok, I see your point. But isn't is better to stick to tf2 instead of using old tf?